### ISSUE TITLE
Support for "pip search" multiple terms.

### ISSUE THREAD

**Issue by wimglenn** [devpi/devpi#560]
pip search word1 word2

Is working fine from warehouse now (the word1 and word2 is considered OR, searching within project name or summary).  The view that handles it should be updated not to raise, but instead let the query go through.  The code which needs to be updated is here:

https://github.com/devpi/devpi/blob/fd1c28ae08f39436883eef8685fcc0648c79d881/web/devpi_web/views.py#L917-L924

**Comment by fschulze** [comment]
The XMLRPC search endpoint on pypi.org is disabled and pip itselfs seems to want to deprecate and remove the search command.

**Comment by wimglenn** [comment]
Yes, but it seems controversial (open since 2018 - https://github.com/pypa/pip/issues/5216) 
And I think they want to remove it due to lack of resources or volunteer efforts to implement it better.  It has problems but I think it should be improved rather than removed.
